R/validate.R

```diff
@@ -8,9 +8,15 @@
 validate_input_values <- function(baseline_scenario, shock_scenario, scenario_geography,
                                   lgd, risk_free_rate, discount_rate, growth_rate,
                                   div_netprofit_prop_coef, shock_year,
-                                  fallback_term, use_company_terms, asset_type) {
+                                  fallback_term, use_company_terms, asset_type,
+                                  settlement_factor = NULL, exp_share_damages_paid = NULL, scc = NULL) {
   input_args <- mget(names(formals()), sys.frame(sys.nframe()))
 
+  if(is.null(settlement_factor) & is.null(exp_share_damages_paid) & is.null(scc)) {
```

Comment:
nitpicking: there should be a lank between the "if" and the "(", please rerun the styler